### PR TITLE
Add /punishment(or /sanction alias)

### PR DESCRIPTION
**Pull request by spekss**
Look at the same of /groups

### CODE REVIEW

File: chat-plugins/info.js [hunk 1]
```diff
@@ -1140,6 +1140,26 @@ exports.commands = {
...
+			"Mute - Unable to talk in the room for 7 minutes" +
+			"Hourmute - Unable to talk in the room for 1 hour" +
+			"Roomban - Banned from the room, until the next restart" +
+			"Balcklist - Banned indefinitely from the room" +
+			"<br \>" +
+			"<b>Global punishment</b>:" +
+			"Lock - Unable to talk in all room and pm. You can just pm the global staff, until after 2 days" +
+			"weeklock - Unable to talk same as lock, but it during 1 week" +
+			"Namelock - Unable to talk a name same as lock for indefinitely" +
+			"Globalban - Banned from Pokemon Showdown, until the next restart"
```
Comment (AustinXII) [inline chat-plugins/info.js]: Ip bans and range lock/bans are also a global punishment

Comment (spekss) [inline chat-plugins/info.js]: ah y i have missing them

**Comment by spekss** [comment]
I have dont add banip beacause it's the same of /global ban ([01:19:55] speks: What is the difference between a global ban and a ban ip?
[01:20:01] @Scyther NO Swiping: nothing)
I'm not sure of a good description for rangelock

**Comment by AustinXII** [comment]
@spekss rangelock locks all users on a certain ip range

**Comment by spekss** [comment]
thanks!

**Comment by Zarel** [comment]
I'm not sure it's necessary to list IP bans and rangelocks here.

File: chat-plugins/info.js [hunk 2]
```diff
@@ -1140,6 +1140,27 @@ exports.commands = {
...
+	punishment: 'sanction',
+	sanction: function (target, rom, user) {
+		if (!this.runBroadcast()) return;
+		this.sendReplyBox(
+			"<b>Room punishment</b>:" +
+			"Warning - show the rules, you have to wait 5 seconds for closed" +
+			"Mute - Unable to talk in the room for 7 minutes" +
+			"Hourmute - Unable to talk in the room for 1 hour" +
+			"Roomban - Banned from the room, it during 1 week" +
+			"Balcklist - Banned for 1 year from the room" +
```
Comment (Zarel) [inline chat-plugins/info.js]: "Balck"

File: chat-plugins/info.js [hunk 3]
```diff
@@ -1140,6 +1140,27 @@ exports.commands = {
 		this.sendReplyBox(`<button name="openSounds" class="button"><i style="font-size: 16px; vertical-align: -1px" class="fa fa-volume-up"></i> Sound</button> (The Sound and Options buttons are at the top right, next to your username)`);
 	},
 
+	'!sanction': true,
+	punishment: 'sanction',
+	sanction: function (target, rom, user) {
```
Comment (Zarel) [inline chat-plugins/info.js]: I would generally call this `punishments`. "sanction" is too obscure of a word for here.

File: chat-plugins/info.js [hunk 4]
```diff
@@ -1140,6 +1140,27 @@ exports.commands = {
...
+		if (!this.runBroadcast()) return;
+		this.sendReplyBox(
+			"<b>Room punishment</b>:" +
+			"Warning - show the rules, you have to wait 5 seconds for closed" +
```
Comment (Zarel) [inline chat-plugins/info.js]: This is not really how English works...

**Comment by sparkychildcharlie** [comment]
Is it just me or will this end up like
``
Warning - Shows the rules, and requires to wait 5 secondsMute - Unable to talk in the room for 7 minutes...
``
since there is no <br /> between each line describing a punishment

**Comment by spekss** [comment]
ah..yes... i have missing  <br /> ..

**Comment by KrisOnPS** [comment]
Why is the title of this /sanction if there isn't even an alias containing sanction?

**Comment by AustinXII** [comment]
@KrisOnPS there was one before obviously

**Comment by spekss** [comment]
Because i have mssing to edit the title of the pr...
anyway add a sanction alias is good idéa, idk why i have removed this

**Pull request closed without merging**